doc/Reference.tex

```diff
@@ -212,8 +212,13 @@ \subsection{inner product $\cdot$}\label{Operation:innerProduct}
 
 
 \subsection{outer product $\otimes$}\label{Operation:outerProduct}
-Computes $z_{i_1,i_2,\ldots,i_{r_1},j_1,\ldots,j_{r_2}} =
-  x_{i_1,,i_2,\ldots,i_{r_1}}y_{j_1,\ldots,j_{r_2}}$.
+Computes 
+\begin{displaymath}
+$z_{i_1,i_2,\ldots,i_{r_1},j_1,\ldots,j_{r_2}} =
+x_{i_1,,i_2,\ldots,i_{r_1}}y_{j_1,\ldots,j_{r_2}}$.
+\end{displaymath}
+where $a$ is the
```

Comment:
There is no a in the above formula.

Reply:
I've noticed a problem with the inner product formula. Not sure when that happened, but it looks like I made the commit :(.